Makefile

```diff
@@ -112,7 +112,9 @@ test-all-valgrind: test-build
 	$(PYTHON) tools/test.py --mode=debug,release --valgrind
 
 test-ci:
-	$(PYTHON) tools/test.py -p tap --logfile test.tap -J parallel sequential message
+	$(PYTHON) tools/test.py -p tap --logfile test.tap --mode=release -J message parallel sequential
```

Comment:
project DBRoulette.xcodeproj in examples/DBRoulette. Next, fill in the values for appKey, appSecret, and root with your app's key, secret, and access type in ‑[DBRouletteAppDelegate application:didFinishLaunchi